packages/theme/components/WishlistSidebar.vue

```diff
@@ -7,14 +7,6 @@
       @close="toggleWishlistSidebar"
       class="sidebar sf-sidebar--right"
     >
-      <template #title>
```

Comment:
Could you explain why was this part deleted?

Reply:
@AdamPawlinski Suggested to remove sidebar-top in wishlist sidebar (on mobile version).
![Screenshot from 2022-03-22 21-30-24](https://user-images.githubusercontent.com/58461801/159959922-cae4a775-caae-4c89-aa5d-c99ec6f09791.png)